README.md

```diff
@@ -1,77 +1,58 @@
-# Crypto LD (Linked Data) _(crypto-ld)_
+# Cryptographic Key Pair Library for Linked Data _(crypto-ld)_
 
 [![Build Status](https://travis-ci.org/digitalbazaar/crypto-ld.png?branch=master)](https://travis-ci.org/digitalbazaar/crypto-ld)
 
-> A Javascript library for cryptographic operations using Linked Data
+> A Javascript library for generating and performing common operations on Linked Data cryptographic key pairs.
 
 ## Table of Contents
 
 - [Background](#background)
 - [Security](#security)
 - [Install](#install)
 - [Usage](#usage)
-- [API](#api-documentation)
 - [Contribute](#contribute)
 - [Commercial Support](#commercial-support)
 - [License](#license)
 
 ## Background
 
-See also (related specs):
+### Supported Key Types
 
-* [Linked Data Proofs 1.0](https://w3c-dvcg.github.io/ld-proofs/)
-* [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/)
+This library provides general Linked Data cryptographic key generation 
+functionality, but does not support any individual key type by default.
 
-As a developer, in order to use this library, you will need to make the 
-following decisions, constrained by your use case:
+To use it, you must [install individual driver libraries](#usage) for each 
+cryptographic key type. The following libraries are currently supported.
 
-1. [Which key type](#choosing-key-type) and suite to use?
-2. What IDs will you give your keys? We recommend the following pattern:
-    `<did or url>#<key fingerprint>`. (See Exporting Key Pair section below
-    for an example of this.)
-3. (Not required, but highly recommended) What is your [Private Key Storage](#private-key-storage) 
-    strategy? (KMS, file system, secure wallet)
+| Type        | Crypto Suite | Library | Usage |
+|-------------|--------------|---------|-------|
+| `ed25519`   | [Ed25519VerificationKey2018](https://w3c-ccg.github.io/ld-cryptosuite-registry/#ed25519) | [`ed25519-key-pair`](https://github.com/digitalbazaar/ed25519-key-pair) | Signatures, VCs, zCaps, DIDAuth |
+| `secp256k1` | [EcdsaSecp256k1VerificationKey2019](https://w3c-ccg.github.io/ld-cryptosuite-registry/#secp256k1) | [`secp256k1-key-pair`](https://github.com/digitalbazaar/secp256k1-key-pair/) | Signatures, VCs, zCaps, DIDAuth, HD Wallets |
+| `rsa`       | [RsaSignature2018](https://w3c-ccg.github.io/ld-cryptosuite-registry/#rsasignature2018) | [`rsa-key-pair`](https://github.com/digitalbazaar/rsa-key-pair) | Legacy/interop signatures and use cases |
```

Comment:
Is it correct to say that `RsaSignature2018` usage is "legacy/interop signatures and use cases". I believe the usage is just the same as `Ed25519VerificationKey2018` or `EcdsaSecp256k1VerificationKey2019`.

Reply:
@gannan08 I removed 'legacy' terminology.